src/tests/components/Toggle.spec.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { fireEvent, render } from "@testing-library/svelte";
+import Toggle from "$lib/components/Toggle.svelte";
+
+describe("Toggle", () => {
+  const props = {
+    checked: false,
+    ariaLabel: "test",
+  };
+
+  it("should render an input checkbox", () => {
+    const { container } = render(Toggle, { props });
+
+    const input = container.querySelector("input") as HTMLInputElement;
+    expect(input).not.toBeNull();
+    expect(input.getAttribute("type")).toEqual("checkbox");
+  });
+
+  it("should render an aria label", () => {
+    const { container } = render(Toggle, { props });
+
+    const input = container.querySelector("input") as HTMLInputElement;
+    expect(input.getAttribute("aria-label")).toEqual(props.ariaLabel);
+  });
+
+  it("should toggle checked", () => {
+    const { component, container } = render(Toggle, { props });
+
+    const input = container.querySelector("input") as HTMLInputElement;
+
+    const onToggle = jest.fn();
+    component.$on("nnsToggle", onToggle);
```

Comment:
Thoughts aloud: probably it's time to switch to `gix` prefix for exposed events...

Reply:
That's a cool idea. I just tried, renaming the events prefix will be easy. On the contrary, migrating the event listener would be quite some work because it is not typed

e.g. `<Back slot="back" on:nnsBack />`

There is no way with `npm run build` or `npm run check` to catch that `nnsBack` would have been renamed to `gixBack`. Only way to catch it automatically is if a particular test `npm run test` has been written.

So feels quite challenging. What do you think?